docs/release-notes/2.37.0.md

```diff
@@ -0,0 +1,108 @@
+# Percona Monitoring and Management 2.37.0
+
+| **Release date:** | April 21, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+
+### Release Highlights1 
+Description of the feature with a link to the documentation
+
+
```

Comment:
```suggestion
### Improved Inventory presentation

**Intro**

This release of PMM starts the series of improvements related to Services and Node presentations inside PMM. The goal will be to provide users with precise and valuable Inventory data of objects added into PMM.

**Problem to solve**

The Inventory in PMM represented the database structure we used to store inventory information and had no additional value for the users. 

Finding and understanding the relationship between components (Services, Nodes, Agents) in PMM and their parameters took a lot of work.

Inventory also was an independent and not connected part of the PMM on UI, while it's the center of all features provided by PMM.

**Current solution** 

We are now releasing only the first step of the improvements on Inventory. The goal is to start the process, show the direction we plan to go with this and start making Inventory the starting point to use and explore other features of the PMM.

\[IMAGE OF NEW INVENTORY, Services tab\]

This first iteration of Inventory will provide the following:

*   Clear _design interface_ for the information. 
*   _Database status_. This information is based on the monitoring metrics. It will let you quickly see the status of the Database when you are working with Inventory.  
*   _Monitoring status p_rovides a summarized status of the monitoring agents related to this Service. Will let users spot problems with Agents and how the PMM monitoring the Service
*   _Detailed information_ provides access to additional parameters and labels of the Services and does not overload the main list simultaneously. 
*   Links to _related_ Service Summary _Dashboards_ or _QueryAnalytics_ results help simplify access to the related PMM sections. 

**What's next**

In the subsequent releases, we are planning to continue improvements in the Invenrory in two extensive directions: 

*   We are improving the presentation of Services and nodes by adding more features like filtering and interconnection between Services and nodes.
*   Structured/clustered presentation of the Services. To let PMM users see their Services organized in clusters and operate on a Cluster level instead of individual Services.

```

Reply:
@catalinaadam PTAL